sql-cli/config/astro-cli.json

```diff
@@ -1,5 +1,9 @@
 {
   "astroCliCompatibility": {
+    "1.10": {
+      "preRelease": false,
+      "sqlCliVersion": ">0.2.2"
+    },
     "1.10.0": {
```

Comment:
:+1: it makes sense to keep this for the current Astro CLI 1.10.0 users since it does not do the full matching. I'm hopeful that from 1.10.1 onwards we'll be able to avoid declaring the versions using the three digits.

Reply:
yes, right! :)